src/siteconfig/forms.py

```diff
@@ -22,16 +22,21 @@ class Meta:
         model = SiteConfig
         # active_semester setting moved to Semester views.
         exclude = ["active_semester"]
+        deck_owner_only = [
```

Comment:
I think just put it as a class variable.  What's the benefit to putting it in Meta? Meta feels like it's for django specific options.  Also you didn't give it the name I asked for!
```
class SiteConfigForm(forms.ModelForm):
     advanced_fields = [...]
```

Access externally with `SiteConfigForm.advanced_fields`

Reply:
fixed